### DISCUSSION TITLE
Add a way to highlight Repos

### DISCUSSION THREAD

**Discussion by showierdata9978** [open-sauced/feedback#31] [Feature request]
I just created a new project in a technology that I have *never* used before. So I was surprised when i could not create a post for It.

**Comment by bdougie** [comment]
Curious what the highlight would be and if you are not able to share a pull request or issue that tells more about the story.

Highlighting a repo sounds interesting, but our product is aimed at encouraging contributions and the main reason why highlights require a link to contribution.

May I suggest writing dev.to post instead? We have been testing that as an alternative

https://docs.opensauced.pizza/features/highlights/

**Comment by showierdata9978** [comment]
Well, even a Commit, as some contributions don't even have a Pull request
attached to it.

**Reply by bdougie** [reply]
Pull requests is the path for making contributions and a better way ti highlight your work. If you are only doing commits to your own project, a blog post would work as well, but I strongly encourage using best practices when showing your work, I.e. pull request.
